### DISCUSSION TITLE
Install of Podman on macOS without using Homebrew

### DISCUSSION THREAD

**Discussion by ARVRHI** [containers/podman#12502] [Q&A]
Is there a Homebrew free way of installing Podman on the Mac that I have missed.

My company does not allow the use of Homebrew on Macs for security reasons. As such I'm looking for a process to get Podman running without having Homebrew on a macOS system. This is especially true for the newer Apple Silicon boxes as well.

Thanks!

**Comment by Luap99** [comment]
You can get `podman-remote-release-darwin.zip` from the release page https://github.com/containers/podman/releases/tag/v3.4.2.
However you would still need to install the dependencies somehow, e.g. `qemu` and `gvproxy`.
